readme.markdown

```diff
@@ -746,6 +746,23 @@ This means that you can safely compile using the compiler when developing your
 software, but you should use Nimble to build the package before publishing it
 to ensure that the dependencies you specified are correct.
 
+To compile code that uses nimble managed packages with `nim`,you need
+to specify `--NimblePath:PATH` option. For example, if your `nimble` directory
```

Comment:
This isn't capitalised AFAIK

Reply:
On the [documentation](https://nim-lang.org/docs/nimc.html#compile-pragma), it's PascalCased, but I think it's case insensitive.